ja_jp/README.adoc

```diff
@@ -0,0 +1,92 @@
+[float]
+image:https://www.zimbra.com/wp-content/uploads/2016/06/zimbra-logo-color-282-1.png[Zimbra,link=https://www.zimbra.com] Zimbra Administration Guide
```

Comment:
This README file contains no specific information related to the Japanese version of the document, so I think it should be deleted, or changed to direct the contributor to look at the repo's README.

The repo's README should describe how to handle translations (Issue #119).